src/python/director/teleoppanel.py

```diff
@@ -447,14 +452,18 @@ def updateConstraints(self):
             elif self.getLFootConstraint() == 'constrained':
                 constraints.extend(ikPlanner.createSixDofLinkConstraints(startPoseName, ikPlanner.leftFootLink, tspan=[1.0, 1.0]))
             elif self.getLFootConstraint() == 'sliding':
-                constraints.extend(ikPlanner.createSlidingFootConstraints(startPoseName)[:2])
+                if not self.getCheckboxState(self.ui.finalPosePlanningOptions):
+                    constraints.extend(ikPlanner.createSlidingFootConstraints(startPoseName)[:2])
```

Comment:
this logic is kind of strange.  Maybe the foot combos should have a new value called 'feet together' instead of overloading the sliding option?